js/main.js

```diff
@@ -222,21 +222,22 @@ var mapView = {
 
   },
   initMap: function() {
-    var self = this;
+    var self = this,
+      mapStyle1 = [ 
```

Comment:
Agreed. I used `mapStyle1` because I was thinking of making multiple style options (which I already did for my forked repo), but I wasn't really paying attention when doing it so yeah, I thought that'd do.